### PR TITLE
Improve fish shell documentation

### PR DESCRIPTION
**Pull request by tommysdk**
* Correct config paths
* Documented jenv init for fish config

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,10 +46,16 @@ Verified on :
...
 	_Same as in previous step, use `~/.profile` on Ubuntu, `~/.zshrc` for Zsh._
 
-    **Fish note**: Instead, copy `~/.jenv/fish/jenv.fish` to `~/.conf/fish/function/jenv.fish`. If you don't have the `export` function, also copy `export.fish`
+    **Fish note**: 
+    Instead, copy `~/.jenv/fish/jenv.fish` to `~/.config/fish/function/jenv.fish`. If you don't have the `export` function, also copy `export.fish`
     ~~~ sh
-        cp ~/.jenv/fish/jenv.fish ~/.conf/fish/function/jenv.fish
-        cp ~/.jenv/fish/export.fish ~/.conf/fish/function/export.fish
+        cp ~/.jenv/fish/jenv.fish ~/.config/fish/function/jenv.fish
+        cp ~/.jenv/fish/export.fish ~/.config/fish/function/export.fish
```
Comment (florianmski) [inline README.md]: The folder should be `functionS` 😄 
I just followed it though and managed to finally make it work, thanks!

**Comment by voronaam** [comment]
The best PR fixing the fish instructions (and an up-to-date one) is this one: PR #307 

This PR should probably be closed in favour of merging PR #307

**Comment by gcuisinier** [comment]
#307 merged

**Comment by tommysdk** [comment]
Thanks for addressing this!

**Pull request closed without merging**